mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -2817,6 +2817,29 @@ func TestAccContainerCluster_nodeAutoprovisioningDefaultsBootDiskKmsKey(t *testi
 	})
 }
 
+func TestAccContainerCluster_nodeAutoprovisioningDefaultsShieldedInstance(t *testing.T) {
+	t.Parallel()
+
+	clusterName := fmt.Sprintf("tf-test-cluster-%s", randString(t, 10))
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:	  func() { testAccPreCheck(t) },
+		Providers:	  testAccProviders,
+		CheckDestroy: testAccCheckContainerClusterDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccContainerCluster_autoprovisioningDefaultsShieldedInstance(clusterName),
+			},
+			{
+				ResourceName:            "google_container_cluster.nap_shielded_instance",
```

Comment:
The test failed with 
```
    provider_test.go:315: Resource specified by ResourceName couldn't be found: google_container_cluster.nap_shielded_instance
```
you specify `google_container_cluster.nap_shielded_instance` here, but have 
```
resource "google_container_cluster" "with_autoprovisioning"
```
in the test config

Reply:
Ah thanks! Will fix :+1: